apps/admin/frontend/src/screens/tally/manual_tallies_form_screen.tsx

```diff
@@ -589,129 +755,168 @@ function ManualResultsDataEntryScreenForm({
                     <TallyInput
-                      autoFocus
-                      id={candidate.id}
-                      data-testid={`${candidate.id}-input`}
-                      value={getValueForInput(candidate.id)}
-                      onChange={(e) =>
-                        updateContestData(candidate.id, e, candidate.name)
-                      }
+                      id="no"
+                      data-testid={`${contest.noOption.id}-input`}
+                      value={getValueForInput('noTally')}
+                      onChange={(e) => updateContestData('noTally', e)}
                     />
-                    <label htmlFor={candidate.id}>
-                      {candidate.name} (write-in)
-                    </label>
-                    <Button
-                      icon="X"
-                      variant="danger"
-                      fill="transparent"
-                      onPress={() => {
-                        removeFormWriteInCandidate(candidate.id);
-                      }}
-                      style={{ marginLeft: 'auto' }}
-                    >
-                      Remove
-                    </Button>
+                    <label htmlFor="no">{contest.noOption.label}</label>
                   </ContestDataRow>
-                ))}
-              </React.Fragment>
-            )}
-            {contest.type === 'yesno' && (
-              <React.Fragment>
-                <ContestDataRow data-testid={`${contest.yesOption.id}`}>
-                  <TallyInput
-                    name="yes"
-                    data-testid={`${contest.yesOption.id}-input`}
-                    value={getValueForInput('yesTally')}
-                    onChange={(e) => updateContestData('yesTally', e)}
-                  />
-                  <label htmlFor="yes">{contest.yesOption.label}</label>
-                </ContestDataRow>
-                <ContestDataRow data-testid={`${contest.noOption.id}`}>
-                  <TallyInput
-                    id="no"
-                    data-testid={`${contest.noOption.id}-input`}
-                    value={getValueForInput('noTally')}
-                    onChange={(e) => updateContestData('noTally', e)}
-                  />
-                  <label htmlFor="no">{contest.noOption.label}</label>
-                </ContestDataRow>
-              </React.Fragment>
-            )}
-            {contest.type === 'candidate' && contest.allowWriteIns && (
-              <AddWriteInRow
-                addWriteInCandidate={(name) => addFormWriteInCandidate(name)}
-                contestId={contest.id}
-                disallowedCandidateNames={disallowedNewWriteInCandidateNames}
-              />
-            )}
+                </React.Fragment>
+              )}
+              {contest.type === 'candidate' && contest.allowWriteIns && (
+                <AddWriteInRow
+                  addWriteInCandidate={(name) => addFormWriteInCandidate(name)}
```

Comment:
nit: style option of just making it `addWriteInCandidate={addFormWriteInCandidate}`

Reply:
3e9944974